akka-http/src/main/scala/akka/http/scaladsl/server/Rejection.scala

```diff
@@ -92,6 +94,15 @@ final case class MalformedHeaderRejection(headerName: String, errorMsg: String,
   extends jserver.MalformedHeaderRejection with RejectionWithOptionalCause
 
 /**
+ * Rejection created by [[akka.http.scaladsl.server.directives.HeaderDirectives.checkSameOrigin]].
+ * Signals that the request was rejected because `Origin` header value is invalid.
+ */
+final case class InvalidOriginHeaderRejection(invalidOrigins: immutable.Seq[SHttpOrigin])
```

Comment:
@nikdon addressed the name change so this was hidden from the web ui.

I'd suggest to use `HttpOriginRange.Default` as the directive param type and change here to
```scala
final case class InvalidOriginHeaderRejection(allowedOrigins: immutable.Seq[SHttpOrigin])
```